test/utils/cleanup.go

```diff
@@ -38,22 +46,94 @@ func CleanupOnInterrupt(cleanup func()) {
 
 // TearDownPipeline will delete created TektonPipeline CRs using clients.
 func TearDownPipeline(clients *Clients, name string) {
-	if clients != nil && clients.Operator != nil {
-		_ = clients.TektonPipeline().Delete(context.TODO(), name, metav1.DeleteOptions{})
+	if clients == nil || clients.Operator == nil {
+		return
+	}
+
+	tc, err := clients.TektonPipeline().Get(context.TODO(), name, metav1.GetOptions{})
```

Comment:
Any reason to not share the `context.Context` all the way instead of creating one each time ?
```go
ctx := context.TODO()
tc, err := clients.TektonPipeline().Get(ctx, …)
# […]
```

Reply:
I will fix that. 🧑‍💻  

these were the lines which just changed in position. So i missed the the repeated context.TODO().